docs/resources/file.md.erb

```diff
@@ -74,10 +74,17 @@ The following examples show how to use this InSpec audit resource.
 ### link_path
 
 The `link_path` property tests if the file exists at the specified path. If the file is a symlink,
-InSpec will resolve the symlink and return the ultimate linked file.
+InSpec will resolve the symlink recursively and return the ultimate linked file.
 
     its('link_path') { should eq '/some/path/to/file' }
 
+### shallow_link_path
+
+The `link_path` property tests if the file exists at the specified path. If the file is a symlink,
```

Comment:
I feel this is an unclear explanation.  A [property](https://www.inspec.io/docs/reference/glossary/#property) doesn't test anything, it returns a value.  Here, you might say, "The `shallow_link_path` property returns the path that the file refers to, only resolving it once (that is, it performs a `readlink` operation).  If the file is not a symlink, `nil` is returned."

Reply:
Fair, updating to reflect that text